charting/action2d/_IndicatorElement.js

```diff
@@ -233,6 +234,19 @@ define(["dojo/_base/lang", "dojo/_base/array", "dojo/_base/declare", "../plot2d/
 					break;
 				}
 			}
+			if(has("dojo-bidi")){
+				i = l - i;
+				r = data[i];
+				//I found by testing that the renderer uses the right side of the arrow pointer to calculate the X position
```

Comment:
I'd probably change the explanation from first person to just explaining the issue, but that's just a minor style note on the comment.